applications/IgaApplication/custom_utilities/brep_topology/brep_trimming_curve.h

```diff
@@ -0,0 +1,73 @@
+#if !defined(KRATOS_BREP_TRIMMING_CURVE_H_INCLUDED )
+#define  KRATOS_BREP_TRIMMING_CURVE_H_INCLUDED
+
+
+// System includes
+
+// External includes
+#include <ANurbs/src/Curve.h>
+
+// Project includes
+#include "iga_application.h"
+#include "iga_application_variables.h"
+
+
+namespace Kratos
+{
+    using namespace ANurbs;
```

Comment:
Maybe dangerous. Everything is imported into the Kratos namespace. You can put the `using` inside the function bodies.